tests/Issues/AnnotationToAttributeRenameAutoImport/Fixture/with_existing_attribute.php.inc

```diff
@@ -20,12 +20,11 @@ class WithExistingAttribute extends AbstractController
 
 namespace Rector\Tests\Issues\RenameAnnotationToAttributeAutoImport\Fixture;
 
-use Symfony\Component\Security\Http\Attribute\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
 
 #[Route(path: '/pro/{id}/networks/{networkId}/sectors', name: 'api_network_sectors', requirements: ['id' => '\d+', 'networkId' => '\d+'])]
-#[IsGranted('TEST')]
+#[\Symfony\Component\Security\Http\Attribute\IsGranted('TEST')]
```

Comment:
@TomasVotruba this is only drawback for this, and run twice will auto-import it, see fixture https://github.com/rectorphp/rector-src/pull/5740/files#diff-fe6ad097b1f7d2f4670de3d6f764aabffb83f16862d8d7893e464012e0be1702

which I think it is ok :)

Reply:
It seems it can be tweaked with original name check, I created new PR to cover it:

- https://github.com/rectorphp/rector-src/pull/5741

Reply:
Looks good to me :)